tests/test_commandline_utils.py

```diff
@@ -1835,6 +1835,36 @@ def test_filter_features_with_drop_blanks():
     assert_frame_equal(df_tsv_output, df_expected)
 
 
+@raises(ValueError)
+def test_filter_features_with_drop_blanks_all_blanks():
```

Comment:
I think this test case will pass when `filter_features.main()` fails on either of the csv/tsv file input. Should we individually check if they both raise ValueError?

Reply:
Good catch! I will add separate tests for CSV/TSV.